Update motivation.md (#43)
sgibson91/cross-stitch-carpentry: index.md

The aim of this lesson is to learn the embroidery technique known as Cross Stitch!

> We will explore cross-stitching as a hobby for programmers and data scientists to bring Mindfulness in our everyday life and boost mental health.

Reviewer: ```suggestion
We will explore cross stitching as a hobby for programmers and data scientists, bringing mindfulness into our everyday life and boosting mental health.
```

> ## Motivation
>

> > To get more background before you get started, please see the [motivation](_extras/motivation.md)

Reviewer: ```suggestion
> To get more background before you begin, please see the [motivation](_extras/motivation.md)
```
